libs/pubsub/query/bench_test.go

```diff
@@ -0,0 +1,84 @@
+package query_test
+
+import (
+	"testing"
+
+	"github.com/tendermint/tendermint/abci/types"
+	"github.com/tendermint/tendermint/libs/pubsub/query"
+	oldquery "github.com/tendermint/tendermint/libs/pubsub/query/oldquery"
+)
+
+const testQuery = `tm.events.type='NewBlock' AND abci.account.name='Igor'`
+
+var testEvents = []types.Event{
```

Comment:
does it make sense to put this in a function local variable and use `b.Run()` subtests?

Reply:
Sure? I don't really see the benefit though. That just moves the test names one level rightward. (I will probably be deleting the PEG cases in a follow-up PR anyway, since we don't really need to keep the old code around—I was thinking to merge it for now just so there would be a clean commit we could point to that got rid of it)